lib/fastlane/setup/setup_ios.rb

```diff
@@ -57,13 +56,30 @@ def copy_existing_files
       end
     end
 
+    def default_generate_appfile
+      # get the proper xcodeproj/workspace and determine the bundle_id
+      # team ID
+      config = {}
+      FastlaneCore::Project.detect_projects(config)
+      project = FastlaneCore::Project.new(config)
+      apple_id = ask_for_apple_id
+      create_appfile(project.default_app_identifier, apple_id)
+    end
+
     def generate_appfile
       Helper.log.info '------------------------------'
       Helper.log.info 'To not re-enter your username and app identifier every time you run one of the fastlane tools or fastlane, these will be stored from now on.'.green
 
       app_identifier = ask('App Identifier (com.krausefx.app): '.yellow)
-      apple_id = ask('Your Apple ID (fastlane@krausefx.com): '.yellow)
+      apple_id = ask_for_apple_id
+      create_appfile(app_identifier, apple_id)
+    end
 
+    def ask_for_apple_id
+      ask('Your Apple ID (fastlane@krausefx.com): '.yellow)
```

Comment:
sorry, not so sure what you mean here. do you mind explaining a bit more? :smile: